panoptes_client/subject_set.py

```diff
@@ -103,6 +104,21 @@ def remove(self, subjects):
 
         return self.links.subjects.remove(subjects)
 
+    def subject_statuses(self, workflow_id):
```

Comment:
keep the naming consistent with the returned resource type
```suggestion
    def subject_workflow_statuses(self, workflow_id):
```